mender-convert-modify

```diff
@@ -113,11 +114,15 @@ if [ "${MENDER_CLIENT_INSTALL}" = "y" ]; then
             log_info "Installing Mender Auth and Mender Update version ${MENDER_CLIENT_VERSION}"
             deb_get_and_install_package mender-auth "${MENDER_CLIENT_VERSION}"
             deb_get_and_install_package mender-update "${MENDER_CLIENT_VERSION}"
+            mender_client_deb_name="${DEB_NAME}"
+
+            log_info "Installing Mender Flash version ${MENDER_FLASH_VERSION}"
```

Comment:
Q: Only for the newest client, or?

Reply:
It is for the new client. All clients 1.x, 2.x, 3.x will go in the previous case `1* | 2* | 3*)`.